lessons/programming/python/conditional-statements/index.mdx

```diff
@@ -126,16 +134,21 @@ invalid score```
 ```
  
 ## 4. Python shorthand if-else:
+Python probvides a shorthand method like the ternary operator of C languge for writing if-else statement in one line.
```

Comment:
Typo:

Python probvides -> **provides** 

ternary operator of C languge -> **language**

statement -> **statements**